src/utils/useHotspot.ts

```diff
@@ -218,11 +218,10 @@ const useHotspot = () => {
       encoded,
     )
 
+    dispatch(connectedHotspotSlice.actions.setConnectedHotspotWifi(undefined))
```

Comment:
The response appears to be null which is fine. Always clear the connected wifi if the write succeeds.